src/test/java/com/dsrm/dsrmbackend/admin/AdminRoomTypeControllerTests.java

```diff
@@ -72,4 +73,27 @@ public void tryToAddNamelessRoomType() throws Exception {
                 .andExpect(status().isBadRequest())
                 .andExpect(jsonPath("$").value("name must not be blank"));
     }
+
+    @Transactional
+    @Test
+    public void deleteExistentRoomType() throws Exception {
+        this.mockMvc.perform(delete("/admin/room-types/2")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNoContent())
+                .andExpect(jsonPath("$.name").value("Sala telewizyjna"));
+        this.mockMvc.perform(get("/room-types/2")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound());
+        this.mockMvc.perform(get("/rooms/3")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.roomType").value(nullValue()));
+    }
+
```

Comment:
It seems to me that checking for the occurrence of a record in the database after a deletion has been performed was supposed to be done using repo and not perform